scripts/watch_txs.js

```diff
@@ -0,0 +1,783 @@
+  "technicalCommitteeMembership"
+);
+
+const txFilters = () => [
+  // Council candidacy submission
+  checkMap(electionsTx("submitCandidacy"), () =>
+    of(`Self-submitted as a council candidate`)
+  ),
+
+  // Council member removal
+  checkMap(
+    electionsTx("removeMember"),
+    ({
+      tx: {
+        args: [member],
+      },
+    }) => of(`Council member removed ${member.toString()}`)
+  ),
+
+  // `set_code` call
+  checkMap(systemTx("setCode"), () => of(`New runtime code was set`)),
+
+  // `set_code_without_checks` call
+  checkMap(systemTx("setCodeWithoutChecks"), () =>
+    of(`New runtime code was set without checks`)
+  ),
+
+  // `set_storage` call
+  checkMap(systemTx("setStorage"), () => of(`Storage was modified directly`)),
+
+  // Democracy proposal seconded
+  checkMap(
+    democracyTx("second"),
+    ({
+      tx: {
+        args: [idx],
+      },
+    }) => of(`Proposal #${idx.toString()} is seconded`)
+  ),
+
+  // `force_transfer` call
+  checkMap(balancesTx("forceTransfer"), ({ tx }) => {
+    const [source, dest, value] = tx.args;
+
+    return of(
+      `\`force_transfer\` was made from ${source} to ${dest} with amount of ${formatDock(
+        value
+      )}`
+    );
+  }),
+
+  // Democracy proposal cancelled
+  checkMap(
+    democracyTx("cancelProposal"),
+    ({
+      tx: {
+        args: [proposalIndex],
+      },
+    }) => of(`Proposal #${proposalIndex.toString()} was cancelled`)
+  ),
+];
+
+const eventFilters = (dock) => {
+  // Emits latest council diff accumulator
+  const councilDiffAccumulator = new ReplaySubject(1);
+  councilDiffAccumulator.next(null);
+
+  return [
+    // Democracy proposal
+    checkMap(
+      both(democracyEvent("Proposed"), democracyTx("propose")),
+      ({
+        tx: {
+          args: [hash],
+        },
+      }) => of(`New Democracy proposal ${hash}`)
+    ),
+
+    // Democracy preimage noted for the proposal
+    checkMap(
+      both(democracyEvent("PreimageNoted"), democracyTx("notePreimage")),
+      (
+        {
+          event: {
+            data: [proposalHash],
+          },
+          tx: {
+            args: [preimage],
+          },
+        },
+        dock
+      ) => {
+        const proposal = dock.api.createType("Call", preimage);
+        const metadata = dock.api.registry.findMetaCall(proposal.callIndex);
+
+        return of(
+          `Proposal preimage is noted for ${proposalHash.toString()}: ${
+            metadata.section
+          }::${metadata.method} with args ${JSON.stringify(
+            proposal.toJSON().args
+          )}`
+        );
+      }
+    ),
+
+    // Democracy proposal fast-tracked
+    checkMap(
+      both(democracyTx("fastTrack"), democracyEvent("Started")),
+      ({
+        tx: {
+          args: [proposalHash],
+        },
+      }) => of(`Proposal is fast-tracked: ${proposalHash.toString()}`)
+    ),
+
+    // Council proposal
+    checkMap(councilEvent("Proposed"), ({ event }, dock) => {
```

Comment:
Event for closing proposal?

Reply:
https://github.com/docknetwork/sdk/pull/304/files#diff-87900fe8bf8aecbf596084810327d179ed7f4c61111e5ce2273449b4120d95aeR431

Reply:
Its redirecting me to `checkMap(democracyTx("second"),`, is that where you intended?

Reply:
Some offset was added with new commits, my apologies
https://github.com/docknetwork/sdk/pull/304/files#diff-87900fe8bf8aecbf596084810327d179ed7f4c61111e5ce2273449b4120d95aeR448

Reply:
Thanks.